jbpm-human-task/jbpm-human-task-workitems/src/test/java/org/jbpm/services/task/wih/HTWorkItemHandlerBaseTest.java

```diff
@@ -608,8 +610,142 @@ public void testTaskWithoutCreatedBy() throws Exception {
     }
     
     @Test
-    public void testTaskWitAutoClaimTaskWithActorAndGroup() throws Exception {
+    public void testTaskWithEnableAutoClaimTaskWithActorAndGroup() throws Exception {
+        TestWorkItemManager manager = new TestWorkItemManager();
+        env.set("Autoclaim", new String("true"));
```

Comment:
I would rather use plain "true" here. It is sufficient.

Reply:
Done